### DISCUSSION TITLE
Play sample with an negative delay

### DISCUSSION THREAD

**Discussion by Fatsplash** [hydrogen-music/hydrogen#1201] [Q&A]
Hi,
I'm playing with doing my own drumkit.
Is there an option to indicate that the beginning of a sample has to be played a little before the moment shown in hydrogen.
For example, if I have a hit with 1s of reverse reverb, can i say i want this sample to be played 1s before the hit in the pattern (whatever the BPM is)
I can't find one but i'm asking to be sure...
Thanks.

**Comment by theGreatWhiteShark** [comment]
Hey @Fatsplash,

I'm not a 100% sure I get you right. You are having a couple of samples and apply to them a specific LADSPA effect that  prepends the audio of the samples with some X frames. Now you want each note corresponding to one of these samples to be triggered X frames prior to its position in the Pattern Editor, am I right?

You can use the Lead & Lag note property of the notes to make them fire earlier. Unfortunately, this feature is only provided at a per-note basis and not for the whole instrument. But using the new editing features of the upcoming version 1.1 in current Github master it won't be too much work either.
